### PR TITLE
How to choose the best operating system

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/computer/operating-system.rst [hunk 1]
```diff
@@ -1,2 +1,50 @@
...
+
+Linux 有很多不同的发行版，所有 Linux 发行版都可以免费获取与安装。
+比较流行的发行版有
+`Ubuntu <https://ubuntu.com/>`__\ 、\
+`Debian <https://www.debian.org/>`__\ 、\
+`Fedora <https://getfedora.org/>`__\ 、\
+`CentOS <https://www.centos.org/>`__ 等。
+
+读者可以根据自己的喜好自行选择合适的 Linux 发行版。
+对于不了解 Linux 的用户，我们推荐安装 XXXX。读者可以进一步参考
```
Comment (seisman) [inline source/computer/operating-system.rst:36]: It's still unclear to me what the best Linux distro is for newbies.

Comment (core-man) [reply source/computer/operating-system.rst:36]: CentOS is dead. 
is Federo a small one? How about Ubuntu? which one has the largest user number?

Comment (core-man) [reply source/computer/operating-system.rst:36]: https://www.tecmint.com/top-most-popular-linux-distributions/

Maybe we can use Ubuntun. I think I will.use it.when I go back to China

Comment (seisman) [reply source/computer/operating-system.rst:36]: I believe ubuntu is the most popular one, but sometimes the packages are too old. Debian and Fedora update more frequently than ubuntu.

Comment (core-man) [reply source/computer/operating-system.rst:36]: In this way, I may choose Fedora.because I am uses to centos

Comment (seisman) [reply source/computer/operating-system.rst:36]: I think i will use fedora, too. But I'm not sure it's suitable for newbies, as each fedora version is maitained for 1 year.

Comment (core-man) [reply source/computer/operating-system.rst:36]: All the three are popular, we can just choose the one we may use in the future. we can write a manual about it just like your centos.

Comment (core-man) [reply source/computer/operating-system.rst:36]: I believe everyone will have a chance or have to choose which is best for them when they get used to linux

Comment (seisman) [reply source/computer/operating-system.rst:36]: Yes, we have to  make a choice, but no hurry.

File: source/computer/operating-system.rst [hunk 2]
```diff
@@ -1,2 +1,51 @@
-操作系统
-========
+选择操作系统
+============
+
+个人电脑有三种常见的操作系统：Microsoft Windows、macOS 和 Linux。
+从市场占有率上看，Microsoft Windows 用户数目远远多于 macOS 和 Linux 用户。
+然而，大多数地震学科研工作者都选择 Linux 或 macOS 作为日常研究的主要操作系统。
```
Comment (core-man) [inline source/computer/operating-system.rst:6]:
```suggestion
然而，大多数地震学科研工作者都选择 macOS 或 Linux 作为日常研究的主要操作系统。
```

File: source/computer/operating-system.rst [hunk 3]
```diff
@@ -1,2 +1,51 @@
...
+主要原因有如下几点：
+
+1. Linux 和 macOS 上可以更容易的获取和使用编程所需的开发工具
```
Comment (core-man) [inline source/computer/operating-system.rst:9]:
```suggestion
1. macOS 和 Linux 上可以更容易的获取和使用编程所需的开发工具
```

File: source/computer/operating-system.rst [hunk 4]
```diff
@@ -1,2 +1,51 @@
...
+2. 大多数地震学相关的程序/代码都是在 Linux 上编写的，且仅在 Linux 和 macOS 下可以使用
+3. Linux 和 macOS 自带了可以对数据做基本处理的命令行工具
```
Comment (core-man) [inline source/computer/operating-system.rst:11]:
```suggestion
3. macOS 和 Linux 自带了可以对数据做基本处理的命令行工具
```

File: source/computer/operating-system.rst [hunk 5]
```diff
@@ -1,2 +1,51 @@
...
+4. 高性能计算服务器几乎都只使用 Linux 系统
+
+对于初学者，我们\ **建议使用 Linux 或 macOS 系统**\ ，除非你非常确定自己的科研
```
Comment (core-man) [inline source/computer/operating-system.rst:14]:
```suggestion
对于初学者，我们\ **建议使用 macOS 或 Linux 系统**\ ，除非你非常确定自己的科研
```

File: source/computer/operating-system.rst [hunk 6]
```diff
@@ -1,2 +1,51 @@
...
+选择操作系统
+============
+
+个人电脑有三种常见的操作系统：Microsoft Windows、macOS 和 Linux。
+从市场占有率上看，Microsoft Windows 用户数目远远多于 macOS 和 Linux 用户。
+然而，大多数地震学科研工作者都选择 Linux 或 macOS 作为日常研究的主要操作系统。
+主要原因有如下几点：
+
+1. Linux 和 macOS 上可以更容易的获取和使用编程所需的开发工具
+2. 大多数地震学相关的程序/代码都是在 Linux 上编写的，且仅在 Linux 和 macOS 下可以使用
```
Comment (core-man) [inline source/computer/operating-system.rst:10]:
```suggestion
2. 大多数地震学相关的程序/代码都是在 Linux 上编写的，且仅在 macOS 和 Linux 下可以使用
```

File: source/computer/operating-system.rst [hunk 7]
```diff
@@ -1,2 +1,51 @@
...
+对于不了解 Linux 的用户，我们推荐安装 XXXX。XXXX 用户可以参考
+《\ :doc:`/computer/linux-setup`\ 》对 Linux 系统进行进一步配置，
+以满足科研工作的需求。
+
+Microsoft Windows
+-----------------
+
+如本节开头所说，不建议地震学科研工作者使用 Windows 操作系统，而建议 Windows 用户
+安装并使用 Linux 系统。Windows 用户使用 Linux 系统的方式有很多，
+比如安装双系统、使用虚拟机安装 Linux，
```
Comment (core-man) [inline source/computer/operating-system.rst:46]:
```suggestion
比如安装双系统、使用虚拟机安装 Linux
```

File: source/computer/operating-system.rst [hunk 8]
```diff
@@ -1,2 +1,51 @@
...
+-----
+
+Linux 有很多不同的发行版，所有 Linux 发行版都可以免费获取与安装。
+比较流行的发行版有
+`Ubuntu <https://ubuntu.com/>`__\ 、\
+`Debian <https://www.debian.org/>`__\ 、\
+`Fedora <https://getfedora.org/>`__\ 、\
+`CentOS <https://www.centos.org/>`__ 等。
+读者可以根据自己的喜好自行选择合适的 Linux 发行版。
+
```
Comment (core-man) [inline source/computer/operating-system.rst:36]:
```suggestion

.. note::

   Red Hat 在 2020 年 12 月宣布将停止 CentOS Linux 项目。CentOS Linux 8 将在
   2021 年末停止更新。同时，考虑到广大用户对 CentOS Linux 7 的使用需求，
   Red Hat 将继续更新和维护其至 2024 年 6 月 30 日。详情请阅读
   `CentOS 通告 <https://lists.centos.org/pipermail/centos-announce/2020-December/048208.html>`__。

```

File: source/computer/operating-system.rst (hunk 2 above)
Comment (seisman) [reply source/computer/operating-system.rst:6]: Why put macos before Linux?

Comment (core-man) [reply source/computer/operating-system.rst:6]: To be consistent with the content. You can also put linux before macOS for the whole 101

Comment (seisman) [reply source/computer/operating-system.rst:6]: I believe most people use Linux, so Linux first.

Comment (core-man) [reply source/computer/operating-system.rst:6]: It is reasonable. please put linux setup before macOS setup.

File: source/computer/operating-system.rst (hunk 8 above)
Comment (seisman) [reply source/computer/operating-system.rst:36]: Do we really want to mention centos 8? centos stream is still promising although i may not use it anymore.

Comment (core-man) [reply source/computer/operating-system.rst:36]: I think we may better let babies kown the announce. you may add that centos linux 8 is replaced by centos stream. We can delete the note when we have a decision about what linux distribution we will use

**Review by core-man — APPROVED** [review]

**Review by core-man — APPROVED** [review]

**Pull request merged**